VirtoCommerce.Domain/Store/Model/Search/SearchCriteria.cs

```diff
@@ -10,7 +11,8 @@ public SearchCriteria()
         }
 
         public string[] StoreIds { get; set; }
-
         public string Keyword { get; set; }
+        public IList<StoreState> StoreStates { get; set; }
```

Comment:
Probably for consistency you could use arrays instead of lists.

Reply:
Fair enough, changed types for new properties from lists to arrays.